### PR TITLE
feat(@vtmn/react): add loader component

### PR DESCRIPTION
**Pull request by gabbloquet**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] Check PR name, it must follow the https://www.conventionalcommits.org pattern
- [x] If it's a new component in CSS, ask for design review

## Description

Loader component for react library. (With csf for storybook story which is compliant with others frameworks)

## Does this introduce a breaking change?

- No

## Other information

❤️ Thank @Decathlon/vitamin-web-administrators !

### CODE REVIEW

File: packages/sources/react/src/components/VtmnLoader/VtmnLoader.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+export interface VtmnLoaderProps extends React.ComponentPropsWithoutRef<'div'> {
+  /**
+   * The size of the loader.
+   * @default 'medium'
+   */
+  size?: VtmnLoaderSize;
+}
+
+export const VtmnLoader = ({
+  size = 'medium',
```
Comment (GaspardMathon) [inline packages/sources/react/src/components/VtmnLoader/VtmnLoader.tsx:15]: Is it possible to use an enumeration to assign the size of the component ?
Like :
```size = VtmnLoaderSize.MEDIUM```

**Review by GaspardMathon — CHANGES_REQUESTED** [review]
Try to use an enumeration

File: packages/sources/react/src/components/VtmnLoader/VtmnLoader.tsx (hunk 1 above)
Comment (gabbloquet) [reply packages/sources/react/src/components/VtmnLoader/VtmnLoader.tsx:15]: Mhhhh this doesn't work like that ;) 
That's an enumeration, not an object

**Review by GaspardMathon — APPROVED** [review]

**Comment by lauthieb** [comment]
Hi @gabbloquet, thanks a lot for your PR! Sorry again for the delay, we had a lot of work last weeks, now I'm available for reviews and help our community packages grow through your contributions 😃 

For me it's okay instead the way you export/import your component. Because of the fact that you import it directly with `import { VtmnLoader } from '@vtmn/react/src/components/VtmnLoader';`, you have a lot of unuseful props showed in your Storybook: 

Capture d’écran 2021-09-21 à 17 29 23

To resolve this, it's very simple. You need to add this line: 
```ts
export { VtmnLoader } from './components/VtmnLoader';
```
inside the file, `packages/sources/react/src/index.ts` sorted alphabetically.

Then, you need to import it in your Storybook like this: 
```ts
import { VtmnLoader } from '@vtmn/react';
```

Thanks a lot again for your contribution and feel free if you need further help!

Laurent

**Review by lauthieb — CHANGES_REQUESTED** [review]
Hi @gabbloquet, thanks a lot for your PR! Sorry again for the delay, we had a lot of work last weeks, now I'm available for reviews and help our community packages grow through your contributions 😃 

For me it's okay instead the way you export/import your component. Because of the fact that you import it directly with `import { VtmnLoader } from '@vtmn/react/src/components/VtmnLoader';`, you have a lot of unuseful props showed in your Storybook: 

Capture d’écran 2021-09-21 à 17 29 23

To resolve this, it's very simple. You need to add this line: 
```ts
export { VtmnLoader } from './components/VtmnLoader';
```
inside the file, `packages/sources/react/src/index.ts` sorted alphabetically.

Then, you need to import it in your Storybook like this: 
```ts
import { VtmnLoader } from '@vtmn/react';
```

Thanks a lot again for your contribution and feel free if you need further help!

_(Bonus, you need to resolve conflicts in `yarn.lock` file, for this feel free to merge our `main branch`, delete `node_modules` & `yarn.lock` files, finally launch `yarn`, with this, it will update dependencies in the same way :))_

EDIT: I just see that you are on vacation, I will merge your PR and change this for you in another commit. Thanks again Gabin!

**Review by lauthieb — APPROVED** [review]

**Pull request merged**